primitives/sr-io/with_std.rs

```diff
@@ -347,18 +349,22 @@ fn with_offchain<R>(f: impl FnOnce(&mut dyn offchain::Externalities) -> R, msg:
 }
 
 impl OffchainApi for () {
+	fn submit_transaction(data: Vec<u8>) -> Result<(), ()> {
```

Comment:
This file is not used anymore. It was probably readded in your big refactoring pr.

https://github.com/paritytech/substrate/pull/4130